x-pack/plugins/rule_registry/server/rule_registry/defaults/field_map.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ecsFieldMap } from '../../generated/ecs_field_map';
+import { pickWithPatterns } from '../field_map/pick_with_patterns';
+
+export const defaultFieldMap = {
```

Comment:
Was thinking about how much of ECS we want to be represented in the root registry in effort to allow increased support of visualizing alerts on generic ECS-based dashboards/visualizations. Added an item to the next Alerts as Data sync to discuss as a group. On the security side we took the approach of adding the entirety of ECS, but will be good to discuss.

Reply:
Yeah, I was trying to keep it lightweight so the types (in the IDE) and autocompletion (in Kibana) are more useful, plus I'd be concerned about people just blindly dumping stuff into ecs fields that shouldn't be there, but I can also see how having all of ECS available by default (although that might have a perf impact on cluster state synchronisation, not sure).